packages/acceptance-tests/pkg-tests-specs/sources/commands/config.test.js

```diff
@@ -57,8 +60,9 @@ function cleanupJsonOutput(output, path) {
     if (typeof setting.default === 'string')
       setting.default = setting.default.replace(pathRegExp, FAKE_WORKSPACE_ROOT);
 
-    if (typeof setting.effective === 'string')
+    if (typeof setting.effective === 'string') {
```

Comment:
Huh, I was pretty sure eslint complained about braces for single-line conditionals, otherwise I'd have put them there myself. My bad, I probably got berry confused with another repo.

Reply:
I'm using the following rule (avoids multiple dedentations at the end of a block):
https://github.com/arcanis/eslint-plugin-arca/blob/master/docs/rules/curly.md

Running `yarn test:lint --fix` should also do the trick

Reply:
Alright, so I'm not losing my mind then, thanks for the intel

> You may want to disable this rule if you don't work with Maël.

😆

Reply:
Alright, so I'm not losing my mind then, thanks for the intel

> You may want to disable this rule if you don't work with Maël.

😆